### ISSUE TITLE
{meta}a does not trigger select all on page or in text areas

### ISSUE THREAD

**Issue by tnrich** [cypress-io/cypress#2915]
Here are two examples of trying to do a "meta/cmd + a" command but nothing actually highlighting. 

```js
it('should ', function() {
    cy.visit("https://docs.cypress.io")
    cy.contains("What you’ll learn")
    cy.get("body").type("{meta}a")
    cy.contains("Our mission")
  })
  it('should ', function() {
    cy.visit("https://bulma.io/documentation/form/textarea/")
    // cy.contains("What you’ll learn")
    cy.get("textarea").first().type("thomas")
    cy.get("textarea").first().type("{meta}a")
  })
```

Does anyone know why would that be? Manually hitting that keyboard combo does highlight stuff on the page. Thanks!

**Comment by jennifer-shehane** [comment]
Hey @tnrich, you're essentially asking for the ability to highlight text on a page. There is an issue open for this here: https://github.com/cypress-io/cypress/issues/2839

**Comment by EtienneBruines** [comment]
*Might* be closed by #311
